democracy_club/apps/core/templatetags/markdown_filter.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from django import template
+from django.utils.safestring import mark_safe
+
+try:
+    from django.utils.encoding import force_text
+except ImportError:
+    from django.utils.encoding import force_unicode as force_text
+
+import markdown
+
+register = template.Library()
+
+
+@register.filter(name="markdown")
+def markdown_filter(value, style="default"):
+    try:
+        return mark_safe(markdown.markdown(value, style))
+    except ImportError:
```

Comment:
This exception will never be raised here, as we're importing Markdown further up in this file. The way to get this working would be to add the `import markdown` line inside the `try` block here.

To be honest though I don't think it's a very useful exception for us to have — without markdown installed we'll get the same sort of exception anyway, so we may as well remove the try/except here